test/run-tests

```diff
@@ -684,24 +705,34 @@ def choose_task(gh, repos, images, config, ansible_id, args):
             statuses = get_statuses(gh, owner, repo, head, args.max_num_statuses)
             commands = get_comment_commands(gh, owner, repo, number)
 
-            for image in images:
-                status_context = get_status_context(
-                    args.variant, image["name"], ansible_id
-                )
-                status = statuses.get(status_context)
+            # We need PR repository to check meta in is_role_distro_support()
+            with checkout_repository(
+                owner, repo, f"pull/{pull}/head"
+            ) as repo_tmp_dir:
+                for image in images:
+                    if is_role_distro_supported(image, repo_tmp_dir):
+                        logging.debug(
+                            f"Unsupported Role. Skipping role {repo} with image {image['source']}."
```

Comment:
I suggest a message like "The role {repo} does not support platform {image['name']}.  Skipping."